Sources/SuperDelegate+LocalNotifications.swift

```diff
@@ -61,31 +63,33 @@ extension SuperDelegate {
     }
     
-    final public func application(application: UIApplication, handleActionWithIdentifier identifier: String?, forLocalNotification notification: UILocalNotification, completionHandler: () -> Void) {
+    @objc(application:handleActionWithIdentifier:forLocalNotification:completionHandler:)
+    final public func application(_ application: UIApplication, handleActionWithIdentifier identifier: String?, for notification: UILocalNotification, completionHandler: @escaping () -> Swift.Void) {
         guard let localNotificationActionCapableSelf = self as? LocalNotificationActionCapable else {
-            noteImproperAPIUsage("Received local notification action but \(self) does not conform to LocalNotificationActionCapable. Ignoring.")
+            noteImproperAPIUsage(text: "Received local notification action but \(self) does not conform to LocalNotificationActionCapable. Ignoring.")
             completionHandler()
             return
         }
         
-        localNotificationActionCapableSelf.handleLocalNotificationActionWithIdentifier(identifier, forLocalNotification: notification, withResponseInfo: nil, completionHandler: completionHandler)
+        localNotificationActionCapableSelf.handleLocalNotification(actionIdentifier: identifier, for: notification, withResponseInfo: nil, completionHandler: completionHandler)
     }
     
-    final public func application(application: UIApplication, handleActionWithIdentifier identifier: String?, forLocalNotification notification: UILocalNotification, withResponseInfo responseInfo: [NSObject : AnyObject], completionHandler: () -> Void) {
+    @objc(application:handleActionWithIdentifier:forLocalNotification:withResponseInfo:completionHandler:)
+    final public func application(_ application: UIApplication, handleActionWithIdentifier identifier: String?, for notification: UILocalNotification, withResponseInfo responseInfo: [AnyHashable : Any], completionHandler: @escaping () -> Swift.Void) {
         guard let localNotificationActionCapableSelf = self as? LocalNotificationActionCapable else {
-            noteImproperAPIUsage("Received local notification action but \(self) does not conform to LocalNotificationActionCapable. Ignoring.")
+            noteImproperAPIUsage(text: "Received local notification action but \(self) does not conform to LocalNotificationActionCapable. Ignoring.")
             completionHandler()
             return
         }
         
-        localNotificationActionCapableSelf.handleLocalNotificationActionWithIdentifier(identifier, forLocalNotification: notification, withResponseInfo: responseInfo as? [String : AnyObject], completionHandler: completionHandler)
+        localNotificationActionCapableSelf.handleLocalNotification(actionIdentifier: identifier, for: notification, withResponseInfo: responseInfo as? [String : Any], completionHandler: completionHandler)
```

Comment:
how would the contents change? You mean it may drop a key/value pair? If it drops something it's dropping a key we weren't capable of handling anyways.